### ISSUE TITLE
About the character mesh

### ISSUE THREAD

**Issue by hellozjj** [nv-tlabs/ASE#2]
Notice there is AMP character (a robot character) in the project. How can I create the paper character for the project? An mjcf files is OK?

**Comment by hellozjj** [comment]
Or will you release the fbx format of the paper character?

**Comment by xbpeng** [comment]
The meshed character in the paper is only for visualization. Simulations are done using the simpler character model.

**Comment by y-zheng18** [comment]
Would you provide the character model? Or how can I animate a character using the predicted motion?

**Comment by xbpeng** [comment]
Sorry, we won't be releasing the meshed character since that was only used for rendering. If you want to use motions produced by the simulation to animate a character, you can add some code to record keyframes from the simulation (i.e. pose of the character at every timestep), and use those to animate your character.

**Comment by taian** [comment]
I know i can export the DoF values to map back to my render mesh/skeleton.
(However, it kinda defeats the purpose of having a physics sim tho.)
Is there a way to swap to different render mesh in Isaac Gym, which shares exactly the same bone hierarchy as the training model. Just want to visualize how it looks on a typical/simple human-like character.

Nice paper and approach btw, inspiring work!
Also, the computed torque/force to be applied at each joint at each timestep, can these values be explicitly known and recorded?

**Comment by xbpeng** [comment]
Yes, there is a way to change the skeleton and mesh by editing the the character's xml. I think you can take a look at some of the examples in IsaacGymEnvs to see how that can be set up:
https://github.com/NVIDIA-Omniverse/IsaacGymEnvs
I have tried doing that myself, so not familiar with the details.

There should also be a way to add a sensor to the env for forces and torques. There might be some examples of those in IsaacGymEnvs as well.

**Comment by mJones00** [comment]
> Sorry, we won't be releasing the meshed character since that was only used for rendering. If you want to use motions produced by the simulation to animate a character, you can add some code to record keyframes from the simulation (i.e. pose of the character at every timestep), and use those to animate your character.

Hello,
is it possible to provide the code for recording the key frames inside the simulation ?
Thanks

**Comment by nicolasfischoeder** [comment]
> Yes, there is a way to change the skeleton and mesh by editing the the character's xml.

It would be really great to train a character with hands and fingers as well. Does anyone have an idea on how to do that? I would like to train a more lifelike model for a different purpose, and definatly require hands.
